repo/plugin.video.nzbdav/resources/lib/router_conn.py

```
@@ -0,0 +1,218 @@
+        data = json.loads(response)
+    except (TypeError, ValueError):
+        return {}
+    return data if isinstance(data, dict) else {}
+
+
+def _xml_root_name(response):
+    """Return the unqualified root XML tag name, lowercased."""
+    import xml.etree.ElementTree as ET  # nosec B405 - trusted service response
+
+    try:
+        root = ET.fromstring(response)  # nosec B314 - trusted service response
```

Ruff (0.15.20)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)